Replace External Data Location

cailafinn opened the pull request:
### Description of work

Add an extra location the testing data can be downloaded from to avoid bottlenecks.

Part of mantidproject/dockerfiles#99

### To test:
Visit the site with a few different hashes and check different files download.
1. http://130.246.80.136/external-data/MD5/98a188e0d125d75453df5663ef68c357


*This does not require release notes* because **nothing user-facing has changed.**

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

cailafinn committed:
Replace external data location

RE mantidproject/dockerfiles#99

sf1919 commented:
This should probably be targetted at main and have v6.12 milestone as it won't get into v6.11 in time.

cailafinn committed:
Make use of floating IP of Load Balancer

We're now linking to a load balancer that links to the node with the external data clone on it. Makes this much less variable when new underlying nodes are used.

RE #99

SilkeSchomann reviewed (approved):
I have modified the link for external data with different hashes and can confirm that the corresponding files are downloaded.
